src/Chapter2.hs

```diff
@@ -741,8 +756,9 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
+contains :: Int -> [[Int]] -> [[Int]]
+contains _ [] = []
+contains n xs = filter (elem n) xs
```

Comment:
You don't need to handle empty lists separately, `filter` perfectly works for them :ok_hand: 

```suggestion
contains n xs = filter (elem n) xs
```

After you remove a separate case, you can eta-reduce this function even further :slightly_smiling_face: